master/docs/manual/configuration/reporters/http_status.rst

```diff
@@ -23,7 +24,7 @@ It requires either `txrequests`_ or `treq`_ to be installed to allow interaction
 
 .. py:class:: HttpStatusPush(serverUrl, user=None, password=None, auth=None, format_fn=None, builders=None, wantProperties=False, wantSteps=False, wantPreviousBuild=False, wantLogs=False, debug=None, verify=None)
```

Comment:
The documentation of the new parameter is missing.

Reply:
Thanks, all done.